Source/Core/Core/HW/WiimoteEmu/Camera.cpp

```diff
@@ -75,19 +75,24 @@ CameraLogic::GetCameraPoints(const Common::Matrix44& transform, Common::Vec2 fie
     const auto point = camera_view * Vec4(v, 1.0);
 
     // Check if LED is behind camera.
-    if (point.z > 0)
-    {
-      // FYI: Casting down vs. rounding seems to produce more symmetrical output.
-      const auto x = s32((1 - point.x / point.w) * CAMERA_RES_X / 2);
-      const auto y = s32((1 - point.y / point.w) * CAMERA_RES_Y / 2);
+    if (point.z < 0)
+      return CameraPoint();
 
-      const auto point_size = std::lround(MAX_POINT_SIZE / point.w / 2);
+    // FYI: Casting down vs. rounding seems to produce more symmetrical output.
+    const auto x = s32((1 - point.x / point.w) * CAMERA_RES_X / 2);
+    const auto y = s32((1 - point.y / point.w) * CAMERA_RES_Y / 2);
```

Comment:
This comment isn't new, but I'd expect both [`std::round`](https://en.cppreference.com/w/cpp/numeric/math/round) and [`std::trunc`](https://en.cppreference.com/w/cpp/numeric/math/trunc) to be symmetric (and a float-int cast [truncates](https://en.cppreference.com/w/cpp/language/implicit_conversion#Floating.E2.80.93integral_conversions)), though [`std::floor`](https://en.cppreference.com/w/cpp/numeric/math/floor)/[`std::ceil`](https://en.cppreference.com/w/cpp/numeric/math/ceil) would be asymetric. Can you clarify what this means (since you originally added it in 4db4840d7c6ca42dd33c4bfc1e46568622e3a96b)?

Reply:
I'll change it to `std::trunc`.